service/core/client/src/main/scala/com/lightbend/lagom/internal/client/CircuitBreakersPanelInternal.scala

```diff
@@ -56,6 +61,21 @@ private[lagom] class CircuitBreakersPanelInternal(
 
   private val createCircuitBreaker = new JFunction[String, Option[CircuitBreakerHolder]] {
 
+    private val allExceptionAsFailure: Try[_] => Boolean = {
+      case _: Success[_] ⇒ false
+      case _             ⇒ true
+    }
```

Comment:
This is the default behaviour which considers all exceptions as failures. This code is copy pasted from the Akka code but in Akka it's a a `private val`

Reply:
Is your editor configured to use Unicode arrows? I've noticed a few slip in. I think we usually try to avoid them in Lagom code.

Reply:
No, this is copy/pasted from Akka code and I didn't review all of them. :-(

Reply:
Ah I see. I wonder if scalariform or scalafmt can auto-correct these.

Reply:
scalafmt supports a [`rewriteTokens`](http://scalameta.org/scalafmt/#0.2.6):

> New flag: --rewriteTokens from1;to1,from2;to2,... to rewrite arbitrary tokens into other tokens. For example, rewrite.tokens = ["⇒;=>,←;<- will rewrite unicode arrows to their ascii equivalents. Flip the tokens to get the opposite behavior. 

scalarifofm only supports rewriting [non-unicode symbols into unicode](https://github.com/scala-ide/scalariform#rewritearrowsymbols) (which is the opposite of what we wanted).